### DISCUSSION TITLE
Why remove the option Prevent WebRTC from leaking local IP addresses

### DISCUSSION THREAD

**Discussion by Pantyhose-X** [uBlockOrigin/uBlock-issues#2087] [Q&A] [answered]
Why

**Comment by hello-smile6** [comment]
Probably needs deleted (Spam)

**Comment by u-RraaLL** [comment, marked as answer]
Because it's no longer needed. The addresses are already obfuscated by the browsers.

**Comment by ghost** [comment]
https://github.com/uBlockOrigin/uBlock-issues/wiki/Dashboard:-Settings#prevent-webrtc-from-leaking-local-ip-address
